bin/soap_mapping.pl

```diff
@@ -144,6 +141,7 @@
 }
 
 if ($output_format =~ /bam/i) {
+	## TODO: WARNING: this won't work anymore and will fail!!!
 	my $cmd2 = "$hadoop jar $sartsam_jar I=$sam_output O=$bam_output SO=coordinate VALIDATION_STRINGENCY=SILENT";
```

Comment:
Here picard is indeed used

Reply:
@robmaz - do we really need this to convert to BAM?

Reply:
I just realized that in the rest of scripts, this lines were commented...and that the output_format was unused. Should I remove also here?